### ISSUE TITLE
Can't translate on image registration panel if we have reloaded different images in the same session

### ISSUE THREAD

**Issue by lauramurgatroyd** [TomographicImaging/iDVC#86]
Tomorrow I will update with specific file types that aren't working for me

**Comment by lauramurgatroyd** [comment]
This is not specific to file type.

If I open the app and load 2 images then register, it's fine.

Straight after if I reload 2 different images and then try to register it doesn't work.

**Comment by lauramurgatroyd** [comment]
I have this when I run from #87 with vtk8.1.2 and viewer current master

**Comment by lauramurgatroyd** [comment]
when we load new images we should run the functionality that "New session" has

**Comment by DanicaSTFC** [comment]
When I manually register the "Translate Z" widget does not update and it is stuck to 0. Is this the problem you mention above or something different?

**Comment by lauramurgatroyd** [comment]
Do you mean just the widget but it allows you to move the images relative to each other?

In my case I could not move the images relative to each other at all, but this only happened if I had loaded an image/different images multiple times int he same session

**Comment by DanicaSTFC** [comment]
> Do you mean just the widget but it allows you to move the images relative to each other?
> 
> In my case I could not move the images relative to each other at all, but this only happened if I had loaded an image/different images multiple times int he same session

Yes, I mean the widget to move the images relative to each other. However I do not know if the shift in the z direction is recorded and passed to the core code correctly. I will check that and open a new issue about it.

**Comment by lauramurgatroyd** [comment]
Did this only happen if you reloaded different images in the same session?

**Comment by DanicaSTFC** [comment]
No, I checked it and it does it all the time for the images I tried.

**Comment by lauramurgatroyd** [comment]
Did we conclude that you did not have a problem with translation on Z in the end?

**Comment by DanicaSTFC** [comment]
Yes, I understood what my concern was and it is not a bug. We just need to clarify that by using the mouse one can sweep the volume in the third direction, this is not a translation. This will be addressed in issue #209

**Comment by DanicaSTFC** [comment]
The problem seems to be that when we reload new data the observers no longer work
https://github.com/TomographicImaging/iDVC/blob/0fed0478d8b14e2c8ded4f6fa82642898dfc8b3a/src/idvc/dvc_interface.py#L1245-L1246
